### ISSUE TITLE
Data Split할 때 어떻게 하는게 합리적일까? (seed 비교해보기 전에 고려해야할 부분)

### ISSUE THREAD

**Issue by MoonJaeHoon** [bcaitech1/p4-dkt-freshtomato#6]
https://github.com/bcaitech1/p4-dkt-freshtomato/blob/f3aff063245e10c1b87721ee12a02f7059fe114d/code/dkt/dataloader.py#L24

일단 신경써줘야할 부분은

1. Train과 Valid의 데이터 특성 및 분포가 비슷해지게끔 split이 되어야 할테니까 Stratified Split을 써줘야할듯 (혹은 Stratified KFold)
(참고 : https://wikidocs.net/43332)
( https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html )

2. 그럼 Stratified Split을 하기 위해 무엇을 Target으로 삼아야할까? (다음이 후보군으로 생각해볼만한 항목들)

    - **진현이횽** : Task의 목적에 맞게, 개인별로 맨마지막 문제를 풀었는가 여부 (0 or 1)
    - **유라쓰** : 문제를 잘 푸는 학생인지, 못 푸는 학생인지 여부 (개인별로 정답률을 도출하여 구분)
    - **민용쓰** : 개인별로 푼 시험의 개수 (testId의 개수) & 시험지 별 난이도(1~9의 난이도가 있으니까, 시험지 난이도 별로 얼마나 풀었는지 => 가중치 둔 평균변수 생성 등)
    - **정현이횽** : 개인별로 푼 문제의 개수 (assessmentItemID의 개수)
    - **재훈** : 개인별로 푼 KnowledgeTag의 개수 혹은 KnowledgeTag의 종류

> 관련해서 피어세션때 얘기해보면 좋을듯
